### ISSUE TITLE
SlackRtmClient event json fails to parse

### ISSUE THREAD

**Issue by jarlfr** [slack-scala-client/slack-scala-client#92]
```
2019-02-19 17:26:42.890 [ERROR] SlackBotTest-akka.actor.default-dispatcher-12 s.r.SlackRtmConnectionActor.[SlackRtmClient] Error reading event: {"type":"desktop_notification","title":"XXX","subtitle":"bot (bot)","msg":"1550593602.003100","ts":"1550593602.003100","content":"pong","channel":"DGBPD4T55","launchUri":"slack:\/\/channel?id=DGBPF4T55&message=1550593602003100&team=RB4C7872O","avatarImage":null,"ssbFilename":"knock_brush.mp3","imageUri":null,"is_shared":false,"event_ts":"1550593602.001000"}
play.api.libs.json.JsResultException: JsResultException(errors:List((/avatarImage,List(JsonValidationError(List(error.expected.jsstring),WrappedArray())))))
	at play.api.libs.json.JsReadable.$anonfun$as$2(JsReadable.scala:25) ~[play-json_2.12-2.6.9.jar:2.6.9]
	at play.api.libs.json.JsError.fold(JsResult.scala:56) ~[play-json_2.12-2.6.9.jar:2.6.9]
	at play.api.libs.json.JsReadable.as(JsReadable.scala:24) ~[play-json_2.12-2.6.9.jar:2.6.9]
	at play.api.libs.json.JsReadable.as$(JsReadable.scala:23) ~[play-json_2.12-2.6.9.jar:2.6.9]
	at play.api.libs.json.JsObject.as(JsValue.scala:124) ~[play-json_2.12-2.6.9.jar:2.6.9]
	at slack.models.package$$anon$7.reads(package.scala:361) ~[slack-scala-client_2.12-0.2.5.jar:0.2.5]
```
(@mccartney edited for formatting)

**Comment by andrey-mikhailov** [comment]
I added a pull request https://github.com/slack-scala-client/slack-scala-client/pull/97 to fix the issue.

**Comment by mccartney** [comment]
Fixed in #97
